apluslms_roman/utils/path_mapping.py

```diff
@@ -5,8 +5,7 @@
 from os import environ
 
 
-logger = logging.getLogger(__name__)
```

Comment:
Keep the `logging.getLogger(__name__)`. It's better as then we can show the python module creating the log entry. Alternative (typically recommended) would be to write `logging.getLogger('apluslms_roman.utils.path_mapping')`, but as `__name__` returns that string, it's better. The recommendation presumes that names and modules might not match 100% and if the file is moved, then the logging name stays the same. I disagree with those arguments and thus prefer `__name__`.

If the file is executable (includes `if __name__ == '__main__'`, then the name for the logger needs to be string, so we don't get logger named `__main__`.

Reply:
Ok, I restore `logging.getLogger( __name__)`  now